testworld/config_test.go

```diff
@@ -0,0 +1,29 @@
+// +build testworld
```

Comment:
From my point of view, this doesn't belong to testworld. Regular integration test and/or functional tests.

Reply:
regular tests wouldn't be sufficient, considering the bug I fixed with urls
functional tests can be done, but this is much more convenient and also verifies things for each pr.

Reply:
also the utils created here are useful for Testworld tests that we do for multi tenancy in the future